107/src/main/java/org/ehcache/jsr107/internal/Jsr107CacheConfigurationParser.java

```diff
@@ -72,11 +81,29 @@ public URI getNamespace() {
 
   @Override
   public Class<Jsr107Service> getServiceType() {
-    return null;
+    return Jsr107Service.class;
   }
 
   @Override
   public Element unparseServiceConfiguration(ServiceConfiguration<Jsr107Service> serviceConfiguration) {
-    throw new UnsupportedOperationException("Not yet implemented");
+    return unparseConfig(serviceConfiguration);
+  }
+
+  @Override
+  public Element createRootElement(Document doc, Jsr107CacheConfiguration cacheConfiguration) {
+    Element rootElement = doc.createElementNS(NAMESPACE.toString(), JSR_NAMESPACE_PREFIX + COLON + MBEANS_ELEMENT_NAME);
+    rootElement.setAttributeNS(XMLConstants.XMLNS_ATTRIBUTE_NS_URI, XMLNS + JSR_NAMESPACE_PREFIX, NAMESPACE.toString());
+    ConfigurationElementState managementState = cacheConfiguration.isManagementEnabled();
+    ConfigurationElementState statisticsState = cacheConfiguration.isStatisticsEnabled();
+    if (ConfigurationElementState.ENABLED == managementState || ConfigurationElementState.DISABLED == managementState) {
+      rootElement.setAttribute(MANAGEMENT_ENABLED_ATTRIBUTE, String.valueOf(cacheConfiguration.isManagementEnabled()
```

Comment:
`managementState.asBoolean()`?

Reply:
If managementState is unspecified then managementState.asBoolean() throws exception